examples/abitest/module/rust/src/lib.rs

```diff
@@ -801,20 +758,26 @@ impl FrontendNode {
     fn test_node_create(&self) -> std::io::Result<()> {
         expect_eq!(
             OakStatus::ERR_INVALID_ARGS,
-            oak::node_create("no-such-contents", self.backend_in[0])
+            oak::node_create("no-such-config", self.backend_in[0])
         );
         expect_eq!(
             OakStatus::ERR_BAD_HANDLE,
             oak::node_create(
-                "backend-code",
+                BACKEND_CONFIG_NAME,
                 oak::ReadHandle {
                     handle: oak::wasm::INVALID_HANDLE
                 }
             )
         );
         let (out_handle, in_handle) = oak::channel_create().unwrap();
-        expect_eq!(OakStatus::OK, oak::node_create("backend-code", in_handle));
-        expect_eq!(OakStatus::OK, oak::node_create("backend-code", in_handle));
+        expect_eq!(
+            OakStatus::OK,
+            oak::node_create(BACKEND_CONFIG_NAME, in_handle)
+        );
+        expect_eq!(
+            OakStatus::OK,
+            oak::node_create(BACKEND_CONFIG_NAME, in_handle)
+        );
```

Comment:
Why create the same thing twice?

Reply:
Just to test that two creations of the same thing work!